lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -1165,7 +1174,9 @@ def main():
             result['Exists']='Connections do exist so we are modifying them'
             if module.check_mode:
                 module.exit_json(changed=True)
```

Comment:
This is essentially lying. Because with nmcli we can't know whether the config changes without actually applying it. Maybe we should declare this module not check mode capable.